zerver/openapi/zulip.yaml

```diff
@@ -8947,6 +8947,165 @@ paths:
+                          the server and included in the response in `presences`. This can be used
+                          as the value of the `presence_last_update_id` parameter when polling
+                          for presence data at the [/users/me/presence](/api/update-presence) endpoint
+                          to tell the server to only fetch the relevant newer data in order to skip
+                          redundant already-known presence information.
+
+                          **Changes**: New in Zulip 9.0 (feature level 263).
+                      server_timestamp:
+                        type: number
+                        description: |
+                          The time when the server fetched the `presences` data included
+                          in the response. This will be not present if the value of `ping_only`
+                          is provided as `true`.
+                      zephyr_mirror_active:
+                        type: boolean
+                        description: |
+                          Whether the mirror bot of the user has been active in the Zephyr
+                          mirror realm. This is present when the organization is a Zephyr
+                          mirror realm.
+                      presences:
+                        type: object
+                        description: |
+                          A dictionary where each entry describes the presence details
+                          of a user in the Zulip organization. This will be not present if
+                          the value of `ping_only` is provided as `true`.
+                        additionalProperties:
+                          description: |
+                            `{user_id}` or `{user_email}`: Object containing the details of a user's
+                            presence. Depending on the value of `slim_presence`, the object's key is
+                            either the user's ID or the user's Zulip API email. If `slim_presence` is
+                            `true`, the presence information is fetched in the modern API format which
+                            includes `active_timestamp` and `idle_timestamp` of the user.
+                          oneOf:
+                            - type: object
+                              additionalProperties:
+                                $ref: "#/components/schemas/Presence"
+                            - type: object
+                              additionalProperties: false
+                              properties:
+                                active_timestamp:
+                                  type: integer
+                                  description: |
+                                    The UNIX timestamp of the last time a client connected to Zulip reported
+                                    that the user was actually present (e.g. via focusing a browser window or
```

Comment:
The content under `oneOf` does not appear in the documentation. I am not sure why this is happening.
![Screenshot 2024-06-07 141053](https://github.com/zulip/zulip/assets/119798962/fd088d3e-f305-4497-93ae-f3f74dbf4ae8)

Reply:
You're going to want to look at `zerver/lib/markdown/api_return_values_table_generator.py` to see how this is being rendered for the API docs, specifically the `render_table` function.

Reply:
I think it is a parsing error. I can try to resolve that in another PR. Is this PR blocked by it?
A similar bug was mentioned [here](https://github.com/zulip/zulip/pull/29937#issuecomment-2110816952).

Reply:
Yes, working on that in a separate PR makes sense. No, this is not blocked on that.

I'm not sure I'll get to a full review today though, as I've got to finish up my work day early for an event. If I don't get to it, know that it's high on my priority list for next week.

Reply:
Yeah we should fix this in a follow-up, as well as cleaning up `/api/get-presence`.